docs/demo/new-bank-account-usecase.md

```diff
@@ -0,0 +1,80 @@
+# New Bank Account Use Case
+
+**Use Case**: As a user, I should be able to open a new bank account online by providing my digital 
+drivers license, digital drivers license assurance/evidence and digital credit report.
+
+## References
+- [Demo Recording](https://www.youtube.com/watch?v=JNUQaOwprT8&feature=youtu.be)
+- [Component Diagram](../images/adapter_component_diagram.svg)
+- [Issuer Adapter](https://github.com/trustbloc/edge-adapter/blob/master/docs/issuer/README.md)
+- [RP Adapter](https://github.com/trustbloc/edge-adapter/blob/master/docs/rp/README.md)
+
+## Components
+
+| Component                          | Cloud URL   | Local URL*                                                  |
+|------------------------------------|-------------|-------------------------------------------------------------|
+| User Wallet                        |             | [Click Here](https://myagent.trustbloc.local/dashboard)     |
```

Comment:
these are local URLs. I understand they should be the testnet URLs if they are to be in this document.

Reply:
I have added column for both testnet/clound and local url. Since we dont have deployed now, its empty. For local,
I have added a note  - https://github.com/trustbloc/edge-sandbox/pull/537/files#diff-316073c863ac197261edb25cfb44d1d3R21.

Reply:
I think we should have both the local sandbox deployment and the testnet sandbox deployment URLs. The table appears to have a placeholder for a "Cloud" URL.